firebase-database/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 19.1.0
+- [feature] The SDK added support for the Firebase Database Emulator. To
```

Comment:
I still say this should be "Added support ..." to match our prior convention...  (else it sounds like the SDK added the emulator support to itself).